docs/.vuepress/components/Icon.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <svg
```

Comment:
Where'd you get all these SVG files? Do you think this is the best approach vs. using an icon library, e.g. [Font Awesome](https://github.com/FortAwesome/vue-fontawesome)? This SVG approach feels like it could get a bit unwieldy if we add more icons down the line.

Reply:
(Commenting for visibility)
These are directly from the figma file, they all share size & visual weight. It might be unwieldy in future I agree but imagine there will be minor updates now that the bulk of this is done (i.e. darkmode handling, passing colors, etc)